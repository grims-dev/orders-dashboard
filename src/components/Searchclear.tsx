import styled from "styled-components";

const Searchclear = styled.div`
  font-size: 18px;
  font-weight: bold;
  line-height: 1.8em;
  text-decoration: underline;
  margin-bottom: 30px;
`;

export default Searchclear;
