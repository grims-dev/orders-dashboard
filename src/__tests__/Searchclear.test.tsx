import { render } from "enzyme";
import * as React from "react";
import toJSON from "enzyme-to-json";
import Searchclear from "../components/Searchclear";

describe("Search clear link renders successfully", () => {
  it("renders without crashing", () => {
    const wrapper = render(<Searchclear />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
