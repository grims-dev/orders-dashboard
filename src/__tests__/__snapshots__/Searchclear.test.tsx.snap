// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search clear link renders successfully renders without crashing 1`] = `
<div
  class="sc-AxjAm bJbhSW"
/>
`;
